pegasus/routes/hoc_routes.rb

```diff
@@ -14,17 +14,17 @@
   JSON.pretty_generate(fetch_hoc_metrics['tutorials'])
 end
 
-# Employee engagement
-get '/api/hour/begin_company/:company' do |company|
-  pass unless DB[:forms].where(kind: 'CompanyProfile', name: company).first
-  pass unless tutorial = DB[:tutorials].where(code: 'codeorg').first
-  launch_tutorial(tutorial, company: company)
-end
```

Comment:
Oops, saw this after merging. I'll add now.

On Wed, Nov 18, 2015 at 6:25 PM, David Bailey <notifications@github.com>
wrote:

> In pegasus/routes/hoc_routes.rb
> <https://github.com/code-dot-org/code-dot-org/pull/5499#discussion_r45292465>
> :
>
> > @@ -14,17 +14,17 @@
> >    JSON.pretty_generate(fetch_hoc_metrics['tutorials'])
> >  end
> >
> > -# Employee engagement
> > -get '/api/hour/begin_company/:company' do |company|
> > -  pass unless DB[:forms].where(kind: 'CompanyProfile', name: company).first
> > -  pass unless tutorial = DB[:tutorials].where(code: 'codeorg').first
> > -  launch_tutorial(tutorial, company: company)
> > -end
>
> It's probably also worth adding a comment that this is left over from HoC
> 2014.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/5499/files#r45292465>.
>



-- 
Tanya Parker
Product Manager, Code.org
+1 (562) 318-4137